### ISSUE TITLE
How to disable variable substitution?

### ISSUE THREAD

**Issue by anisalibegic** [drwatson1/dbup-cli#20]
Currently, any text that's surrounded by `$` is treated as a variable and it's content is replaced with the text defined in config/env file. This causes a problem when having $function$ or $body$ in PostgreSQL script files. Those are not variables yet they are treated like they are.

Current workaround is using...
```yml
vars:
  body: $body$
  functions: $function$
  ...
```
...which entirely depends on a developer and it's way of writing scripts.

Is it possible to disable variable substitution with `dbup.yml` configuration file like we can when using a package in .NET? Something like `vars: no` or when vars list is empty then disable substitution.

I searched through source code and found no usages of `WithVariablesDisabled()` method.

**Comment by huesie** [comment]
I'm sure you're aware - and as a workaround we use ```$$``` when using DbUp-CLI:
```
CREATE OR REPLACE FUNCTION MyFunc() RETURNS integer
LANGUAGE "plpgsql"
AS $$
BEGIN
  ...snip...;
END;
$$;
```
I appreciate that all of your developers would need to follow this + nesting is tricky.

**Comment by anisalibegic** [comment]
Yes, of course. We can't use this approach because our database has already hundreds of functions and it's not convenient to modify them all. Is it possible to add a `DisableVars` parameter to the config file? Perhaps I could wire up a quick PR to fix that...

**Comment by huesie** [comment]
As an 'extra pair of eyes' I've tested the proposed ```disableVars: yes``` and it worked perfectly for me - it's up to @drwatson1 if he wants to accept this approach

**Comment by drwatson1** [comment]
I've tested the addition and it works as expected. PR looks good too.

@anisalibegic Thank you for the report and good solution
@huesie Thank you for testing
